fluss-common/src/test/java/com/alibaba/fluss/utils/PartitionUtilsTest.java

```diff
@@ -17,47 +17,76 @@
 package com.alibaba.fluss.utils;
 
 import com.alibaba.fluss.config.AutoPartitionTimeUnit;
+import com.alibaba.fluss.exception.PartitionSpecInvalidException;
 
 import org.junit.jupiter.api.Test;
 
 import java.time.LocalDateTime;
 import java.time.ZoneId;
 import java.time.ZonedDateTime;
+import java.util.Arrays;
+import java.util.Collections;
 
+import static com.alibaba.fluss.utils.PartitionUtils.generateAutoPartitionName;
+import static com.alibaba.fluss.utils.PartitionUtils.getPartitionName;
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
-/** Test for {@link com.alibaba.fluss.utils.AutoPartitionUtils}. */
-class AutoPartitionUtilsTest {
+/** Test for {@link PartitionUtils}. */
+class PartitionUtilsTest {
 
     @Test
-    void testGetPartitionString() {
+    void testGetPartitionName() {
+        assertThat(getPartitionName(Collections.singletonList("a"), Collections.singletonList("1")))
+                .isEqualTo("1");
+
+        assertThat(getPartitionName(Arrays.asList("a", "b"), Arrays.asList("1", "2")))
+                .isEqualTo("1$2");
+
+        assertThatThrownBy(
+                        () ->
+                                getPartitionName(
+                                        Arrays.asList("a", "b", "c"), Arrays.asList("1", "2")))
+                .isInstanceOf(IllegalArgumentException.class)
+                .hasMessageContaining(
+                        "The number of partition keys and partition specs should be the same.");
+
+        assertThatThrownBy(
+                        () -> getPartitionName(Arrays.asList("a", "b"), Arrays.asList("$1", "2")))
```

Comment:
Personally, I think the `getPartitionName` should just does getter/generate thing. The validation should be done in a separate method. This will make logic cleaner and make the `getPartiionName` better performance.

Reply:
Done!